### PR TITLE
[FXLA11Y] Re-adding font section and adding WCAG success criteria

### PR DESCRIPTION
**Pull request by wareid**
As discussed in the last meeting, this PR re-adds the font selection recommendations. This PR also begins the work of restructuring the document to include the success criteria as they are in the legibility section.

### CODE REVIEW

File: epub33/fxl-a11y/index.html
Comment (mattgarrish) [inline epub33/fxl-a11y/index.html]: I'm not sure it's accurate to say there isn't a font size requirement. 1.4.4 says that text needs to be able to resize to 200% of its default size without loss of readability or functionality.

Reading systems that run on tablets or similar devices should let you zoom fxl pages, but desktop-based apps don't, at least not when I last checked (which was some time back). So long as the text scales with the page on the RSes that do zoom, this SC won't be an issue. You can expect that users who need the functionality will seek out those RSes. But if reading systems don't scale the text, and you can't modify the text properties as a user, then there'd be a problem.

(This also leads into the ever problematic reflow SC where zooming fxls doesn't reflow the text as it enlarges, so you end up having to scroll to read lines.)

Comment (mattgarrish) [reply epub33/fxl-a11y/index.html]: And to be clear, I'm not expecting this to be changed before you merge. Just wanted to mention it.

**Comment by gregoriopellegrino** [comment]
Thanks for the great work done and sorry for the late analysis: I have had a bad flu the past few days.

The section on readability -- net of Matt's comments -- looks fine to me. Perhaps we could add a note to section _2.5.4 Font selection_ to say that this section is not normative, even though the whole document is not, but I would like to say that these are recommendations, not tied to international guidelines.

With respect to the linkage to the WCAG Success Criteria, it seems to me a very good job. I had also envisioned a reorganization of the content, following the structure of the SCs:

1. Perceivable
1.1 Text Alternatives
1.2 Time-based Media
1.3 Adaptable
1.4 Distinguishable
2. Operable
2.1 Keyboard Accessible
2.2 Enough Time
2.3 Seizures and Physical Reactions
2.4 Navigable
2.5 Input Modalities
3. Understandable
3.1 Readable
3.2 Predictable
3.3 Input Assistance
4. Robust
4.1 Compatible

But maybe this is a bit too much, I would leave it to the group discussion.

In the meantime, thank you for the work!

**Review by gregoriopellegrino — APPROVED** [review]

File: epub33/fxl-a11y/index.html
Comment (jonaslil) [reply epub33/fxl-a11y/index.html]: The default font size in browsers is 16px, not 16pt, as far as I know.

Another thing I thought about:

If the RS allows scaling the page, the text size in a fixed layout book will be influenced by two things, when read on a smaller screen:

1. font size setting
1. page size and/or line length

The larger the page, the smaller the text will be when the page is scaled down to fit the viewport. If the lines flow out of the visible area, so that you have to scroll horizontally, the text will be very difficult to read.

**Comment by wareid** [comment]
See:

* For EPUB Fixed Layout Accessibility:
    * [Preview](https://cdn.statically.io/gh/w3c/epub-specs/fxl-editorial-3/epub33/fxl-a11y/index.html)
    * [Diff](https://services.w3.org/htmldiff?doc1=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://w3c.github.io/epub-specs/epub33/fxl-a11y/index.html&doc2=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://cdn.statically.io/gh/w3c/epub-specs/fxl-editorial-3/epub33/fxl-a11y/index.html)

**Review by gregoriopellegrino — APPROVED** [review]
Great improvement for me, thanks!

**Pull request merged**
